CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
-# Contributing to Internal Computing and Calculus for Advanced Statistics Resources
+# Contributing
 
-These are recommendations when contributing to the contents of the internal `CCAS` (Computing and Calculus for Advanced Statistics) repository.
+These are recommendations when contributing to the contents of the internal repository.
```

Comment:
Maybe add some TODOs with course abbreviation placeholders that users are auppoaed to replace?

Reply:
solved